Level-2/trmv/trmv.cpp

```diff
@@ -0,0 +1,145 @@
+/* ************************************************************************
+ * Copyright (C) 2019-2022 Advanced Micro Devices, Inc. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell cop-
+ * ies of the Software, and to permit persons to whom the Software is furnished
+ * to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IM-
+ * PLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNE-
+ * CTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * ************************************************************************ */
+
+#include "helpers.hpp"
+#include <complex>
+#include <hip/hip_runtime.h>
+#include <math.h>
+#include <rocblas/rocblas.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <vector>
+
+int main(int argc, char** argv)
+{
+
+    typedef std::complex<float> T;
+
+    helpers::ArgParser options("Nx");
+    if(!options.validArgs(argc, argv))
+        return EXIT_FAILURE;
+
+    rocblas_status rstatus = rocblas_status_success;
+
+    rocblas_int N    = options.N;
+    rocblas_int incx = options.incx;
+
+    // Pre-filled parameters
+    const rocblas_fill uplo = rocblas_fill_lower;
+    const rocblas_diagonal diag = rocblas_diagonal_non_unit;
+
+    //trans is fixed to rocblas_operation_none in this example and support for other options would be added in the future release
+    const rocblas_operation trans = rocblas_operation_none;
+
+    size_t sizeX, absIncx;
+
+    rocblas_int lda   = N;
+    int      sizeA = lda * int(N);
```

Comment:
This is inconsistent with other sizes, should be size_t or int64_t result and casting.